scripts/configure.js

```diff
@@ -643,7 +643,7 @@ const getConfig = (() => {
             Podfile: join(
               `require_relative '${testAppRelPath}/test_app'`,
               "",
-              "use_flipper! false if ENV['USE_FLIPPER'] == '0'",
+              "use_flipper! false unless ENV['USE_FLIPPER']",
```

Comment:
Can we make an explicit check in case people are already setting `USE_FLIPPER=0`:
```suggestion
              "use_flipper! false unless ENV['USE_FLIPPER'] == '1'",
```